### PR TITLE
fix showing optional params in signature

### PR DESCRIPTION
**Pull request by keertip**

### CODE REVIEW

File: lib/dartdoc.dart [hunk 1]
```diff
@@ -99,8 +99,9 @@ class DartDoc {
     SourceFactory sourceFactory =
         new SourceFactory(/*contentCache,*/ resolvers);
 
-    var options = new AnalysisOptionsImpl()..analyzeFunctionBodies = false;
-    options.cacheSize = 512;
+    var options = new AnalysisOptionsImpl()
+      ..analyzeFunctionBodies = false
+      ..cacheSize = 512;
```
Comment (sethladd) [inline lib/dartdoc.dart]: nice.

File: lib/src/model.dart [hunk 2]
```diff
@@ -357,7 +357,7 @@ abstract class ModelElement {
 
     String renderParams(Iterable<Parameter> params,
         [String open = '', String close = '']) {
```
Comment (sethladd) [inline lib/src/model.dart]: would love if these were named params. makes the call site much easier to read.

**Comment by sethladd** [comment]
Thanks!

File: lib/src/model.dart (hunk 2 above)
Comment (keertip) [inline lib/src/model.dart]: done.

**Pull request merged**